src/styles/search.css

```diff
@@ -0,0 +1,41 @@
+.search-form {
+  align-items: center;
+  background-color: #fff;
+  border-radius: var(--border-rad);
+  box-shadow: 0 0 1px 1px var(--nasa-red);
+  display: flex;
+  height: 2rem;
+  margin-top: var(--white-space);
+  position: relative;
+  width: 100%;
+}
+
+.search-form__input {
+  background-color: rgba(0, 0, 0, 0);
+  border: none;
+  border-radius: var(--border-rad);
+  font-size: 20px;
+  height: inherit;
+  inset: 0 0 0 0;
+  min-width: 0;
+  padding: 0 3rem 0 0.5rem;
+  position: absolute;
+  width: 100%;
```

Comment:
A lot of these properties are not actually doing anything, it would be good to go through them and see how many you can remove. I usually do this in the dev tools in the browser, turning off properties I think can be removed and seeing if there is a difference in behaviour. A quick run through that and I could take about about half the styles in this file without issue